src/main/webapp/app/overview/course-exercises/course-exercises.component.html

```diff
@@ -3,14 +3,20 @@
         <div class="mt-3 position-relative" *ngIf="course">
             <div class="exercise-controls" *ngIf="course!.exercises && course!.exercises.length > 0">
                 <div class="d-flex align-items-center mr-2">
-                    <label class="d-flex align-items-center mb-0 mr-2">
-                        <input (change)="toggleFilters([filterType.OVERDUE])" [checked]="activeFilters.has(filterType.OVERDUE)" type="checkbox" />
-                        <span class="ml-2" jhiTranslate="artemisApp.courseOverview.exerciseList.overdue">Hide overdue</span>
-                    </label>
-                    <label class="d-flex align-items-center mb-0 mr-2">
-                        <input (change)="toggleFilters([filterType.NEEDS_WORK])" [checked]="activeFilters.has(filterType.NEEDS_WORK)" type="checkbox" />
-                        <span class="ml-2" jhiTranslate="artemisApp.courseOverview.exerciseList.fullScore">Hide full score</span>
+                    <label class="d-flex align-items-center mb-0 mr-2" *ngIf="course.isAtLeastTutor">
+                        <input (change)="toggleFilters([filterType.UNRELEASED])" [checked]="activeFilters.has(filterType.UNRELEASED)" type="checkbox" />
+                        <span class="ml-2" jhiTranslate="artemisApp.courseOverview.exerciseList.unreleased">Hide unreleased</span>
                     </label>
```

Comment:
A small component for this can make everything cleaner, since it gets repeated 3 times in your changes.

Reply:
I agree with you, but we will address this later in a refactoring of the UI in this component